flake.nix

```diff
@@ -0,0 +1,254 @@
+{
+  description = "Emacs versions for CI";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs/nixpkgs-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+
+    flake-compat = { url = "github:edolstra/flake-compat"; flake = false; };
+    "emacs-23.4" = { url = "https://ftp.gnu.org/gnu/emacs/emacs-23.4.tar.gz"; flake = false; };
```

Comment:
I think these URLs should be prefixed with `tarball+https` scheme. See https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix3-flake.html#types

Reply:
Oh, that could make sense, just to be explicit. Though the doc you linked to says

> if the extension corresponds to a known archive format (.zip, .tar, .tgz, .tar.gz, .tar.xz, .tar.bz2 or .tar.zst), then the tarball+ can be dropped.

Reply:
That's right. Thanks.